server/lib/device/device.saveStringState.js

```diff
@@ -29,12 +36,6 @@ async function saveStringState(device, deviceFeature, newValue) {
     },
   );
 
-  deviceFeature.last_value_string = newValue;
-  deviceFeature.last_value_changed = new Date();
-
-  // save local state in RAM
-  this.stateManager.setState('deviceFeature', deviceFeature.selector, deviceFeature);
```

Comment:
I don't think we want to change the order this is done, the goal is to update the state in RAM only after the DB is updated

Reply:
Because this is how it is done in saveState and saveHistoricalState

Reply:
Oh indeed, you're right! Good for me then